src/globus_sdk/scopes/data/flows.py

```diff
@@ -0,0 +1,117 @@
+from __future__ import annotations
+
+import typing as t
+
+from globus_sdk._types import UUIDLike
+
+from ..builder import ScopeBuilder, ScopeBuilderScopes
+
+
+class _FlowsScopeBuilder(ScopeBuilder):
+    """
+    The Flows Service breaks the scopes/resource server convention:
+      its resource server is a domain name but its scopes are built around the client id
+    Given that there isn't a simple way to support this more generally
+      (and we shouldn't encourage supporting this more generally), this class serves to
+      build out the scopes accurately specifically for Flows
```

Comment:
Below, another private class uses full sentences so I'm recommending the same here.

```suggestion
      build out the scopes accurately specifically for Flows.
```

Reply:
:+1: This was a lift-and-shift, but I don't object to tweaking this while we're here.